datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxUpgradeClusterConverter.java

```diff
@@ -28,6 +27,9 @@ public UpgradeV4Request sdxUpgradeRequestToUpgradeV4Request(SdxUpgradeRequest sd
         upgradeV4Request.setRuntime(sdxUpgradeRequest.getRuntime());
         upgradeV4Request.setDryRun(sdxUpgradeRequest.isDryRun());
         upgradeV4Request.setLockComponents(sdxUpgradeRequest.getLockComponents());
+        if (Objects.nonNull(sdxUpgradeRequest.getShowAvailableImages())) {
+            upgradeV4Request.setShowAvailableImages(UpgradeShowAvailableImages.valueOf(sdxUpgradeRequest.getShowAvailableImages().name()));
```

Comment:
@pdarvasi will add a unit test in a separate pr to check the 2 enum has the same values